.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Happy to remove this and not start a nitpick battle, but I think it's a good practice for an open source project to maintain its own `.gitignore` for files that are generated in the course of development on it. :smile:

Consider it gone.